assets/tuned/stalld/Makefile

```diff
@@ -33,14 +43,16 @@ install:
 	$(INSTALL) -m 755 -d $(DESTDIR)$(LICDIR)/$(NAME)
 	$(INSTALL) gpl-2.0.txt -m 644 $(DESTDIR)$(LICDIR)/$(NAME)
 
+
 .PHONY: clean tarball redhat push
 clean:
 	@test ! -f stalld || rm stalld
 	@test ! -f stalld-static || rm stalld-static
 	@test ! -f src/stalld.o || rm src/stalld.o
 	@test ! -f $(TARBALL) || rm -f $(TARBALL)
 	@make -C redhat clean
```

Comment:
Not happy with this redhat line

```
$ make clean
make[1]: *** redhat: No such file or directory.  Stop.
make: *** [Makefile:53: clean] Error 2
```

Reply:
Again, NTO ships with the bare minimum to make `stalld` in NTO.